build_scripts/windows/scripts/build.cmd

```diff
@@ -137,18 +144,27 @@ if %errorlevel% neq 0 goto ERROR
 
 pushd %BUILDING_DIR%
 %BUILDING_DIR%\python.exe %REPO_ROOT%\scripts\compact_aaz.py
+if %errorlevel% neq 0 goto ERROR
 %BUILDING_DIR%\python.exe %~dp0\patch_models_v2.py
+if %errorlevel% neq 0 goto ERROR
 %BUILDING_DIR%\python.exe %REPO_ROOT%\scripts\trim_sdk.py
+if %errorlevel% neq 0 goto ERROR
 popd
 
 REM Remove pywin32 help file to reduce size.
 del %BUILDING_DIR%\Lib\site-packages\PyWin32.chm
 
 echo Creating the wbin (Windows binaries) folder that will be added to the path...
 mkdir %BUILDING_DIR%\wbin
-copy %REPO_ROOT%\build_scripts\windows\scripts\az.cmd %BUILDING_DIR%\wbin\
-copy %REPO_ROOT%\build_scripts\windows\scripts\azps.ps1 %BUILDING_DIR%\wbin\
-copy %REPO_ROOT%\build_scripts\windows\scripts\az %BUILDING_DIR%\wbin\
+if "%TARGET%"=="msi" (
+    copy %REPO_ROOT%\build_scripts\windows\scripts\az.cmd %BUILDING_DIR%\wbin\
+    copy %REPO_ROOT%\build_scripts\windows\scripts\azps.ps1 %BUILDING_DIR%\wbin\
+    copy %REPO_ROOT%\build_scripts\windows\scripts\az %BUILDING_DIR%\wbin\
+) else (
+    copy %REPO_ROOT%\build_scripts\windows\scripts\az.cmd %BUILDING_DIR%\wbin\
+    powershell -Command "(Get-Content '%BUILDING_DIR%\wbin\az.cmd') -replace 'SET AZ_INSTALLER=MSI','SET AZ_INSTALLER=ZIP' | Set-Content '%BUILDING_DIR%\wbin\az.cmd'"
```

Comment:
This string manipulation can be fragile if the contents of `%BUILDING_DIR%\wbin\az.cmd` changes. I suggest we create a separate `az_zip.cmd`, `az_msi.cmd` and rename then to `az.cmd` when packaging.